BusinessTripApplication/Web.config

```diff
@@ -82,7 +82,7 @@
     </compilers>
   </system.codedom>
   <connectionStrings>
-    <add name="UserContext" connectionString="Data Source=(LocalDB)\MSSQLLocalDB;Integrated Security=True" providerName="System.Data.SqlClient" />
+    <add name="BusinessContext" connectionString="Data Source=(LocalDB)\MSSQLLocalDB;Integrated Security=True" providerName="System.Data.SqlClient" />
```

Comment:
You should resolve the conflicts regarding the name of the db: please rename BussinesContext everywhere in your project ( it should be DatabaseContext)